scripts/run_lunarlander_continuous.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+"""Train or test baselines on LunarLanderContinuous-v2.
+
+- Author: Curt Park
+- Contact: curt.park@medipixel.io
+"""
+
+import argparse
+import importlib
+
+import gym
+import numpy as np
+import torch
+
+# configurations
+parser = argparse.ArgumentParser(description="Pytorch RL baselines")
```

Comment:
It can be at a later commit, but can we discuss parameter order of `parser.add_argument()`?

Also, "--test" and "--log" do not need `dest` parameter.

Reply:
Thanks, applied your feedback.